src/applications/disability-benefits/2346/components/Accessories.jsx

```diff
@@ -74,32 +74,29 @@ class Accessories extends Component {
         )}
         {!haveAccessoriesBeenOrderedInLastTwoYears &&
           !areAccessorySuppliesEligible && (
-            <AlertBox
-              headline="You can't add accessories to your order at this time"
-              content={
-                <div className="accessories-two-year-alert-content">
-                  <p>
-                    You can only order accessories that you've received in the
-                    past 2 years.
-                  </p>
-                  <p>
-                    If you need accessories like domes, wax guards, cleaning
-                    supplies, or dessicant, call the DLC Customer Service
-                    Section at{' '}
-                    <a
-                      aria-label="3 0 3. 2 7 3. 6 2 0 0."
-                      href="tel:303-273-6200"
-                    >
-                      303-273-6200
-                    </a>{' '}
-                    or email{' '}
-                    <a href="mailto:dalc.css@va.gov">dalc.css@va.gov</a>.
-                  </p>
-                </div>
-              }
-              status="info"
-              isVisible
-            />
+            <va-alert status="info" visible>
+              <h3 slot="headline">
+                You can’t add accessories to your order at this time
+              </h3>
+              <div className="accessories-two-year-alert-content">
+                <p>
+                  You can only order accessories that you’ve received in the
+                  past 2 years.
+                </p>
+                <p>
+                  If you need accessories like domes, wax guards, cleaning
+                  supplies, or dessicant, call the DLC Customer Service Section
+                  at{' '}
+                  <a
+                    aria-label="3 0 3. 2 7 3. 6 2 0 0."
+                    href="tel:303-273-6200"
+                  >
+                    303-273-6200
+                  </a>{' '}
```

Comment:
This can be replaced with the `<va-telephone>` web component - https://design.va.gov/storybook/?path=/docs/components-va-telephone--default

Reply:
I'll be opening a separate ticket for some of these other linting errors! Trying to compartmentalize changes.